File: CS/Dashboard_PrevYearData/Form1.cs
using System.Windows.Forms;
using DevExpress.XtraBars.Ribbon;

namespace Dashboard_PrevYearData {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
            dashboardDesigner1.CreateRibbon();
            dashboardDesigner1.LoadDashboard(@"..\..\Data\PrevYearData.xml");
        }
    }
}

File: VB/Dashboard_PrevYearData/Form1.vb
Imports System.Windows.Forms

Namespace Dashboard_PrevYearData

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
            dashboardDesigner1.CreateRibbon()
            dashboardDesigner1.LoadDashboard("..\..\Data\PrevYearData.xml")
        End Sub
    End Class
End Namespace
